frontend/src/component/context/ContectFormChip/ContextFormChipList.tsx

```diff
@@ -6,9 +6,16 @@ const StyledContainer = styled('ul')(({ theme }) => ({
     display: 'flex',
     flexWrap: 'wrap',
     gap: theme.spacing(1),
-    padding: 0,
+    padding: theme.spacing(2),
     margin: 0,
     marginBottom: '1rem !important',
+    maxHeight: '412px',
+    overflow: 'auto',
```

Comment:
what do we get with a large number of items? what does it look like? a regular scrollbar?

Reply:
yes exactly